app/templates/partials/tasklist_section.html

```diff
@@ -9,11 +9,11 @@ <h2 class="app-task-list__section">
       {% for question_page in section["questions"]%}
       <li class="app-task-list__item">
         <span class="app-task-list__task-name">
-          <a href="/continue_application/{{application_id}}?section_name={{section["section_name"]}}&page_name={{question_page["question"]}}" aria-describedby="page-title">
-            <p id="page-title" class="govuk-body">{{question_page["question"]}}</p>
+          <a class="govuk-body govuk-link govuk-link--no-visited-state" href="/continue_application/{{application_id}}?section_name={{section["section_name"]}}&page_name={{question_page["question"]}}">
```

Comment:
Can we use `url_for` to build this route?

Reply:
i think we can actually, from the docs: Variable arguments that are unknown to the target endpoint are appended to the generated URL as query arguments. Ill give it a go